lib/compose-tweet.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+
+module.exports = (commit, standard) => {
+  let [commitTitle] = commit.message.split("\n");
+  for (const exclude of ["Editorial", "Meta", "Review Draft Publication"]) {
+    if (commitTitle.startsWith(`${exclude}:`)) {
+      return null;
+    }
+  }
+
+  const authorEmails = standard.authors.map(author => author.email);
+  if (!authorEmails.includes(commit.author.email)) {
+    commitTitle += ` (thanks ${commit.author.name}!)`;
+  }
+
+  return `${commitTitle}\n${commit.url}`;
```

Comment:
```suggestion
  return `${commitTitle}\n\n${commit.url}`;
```

?

Reply:
Or maybe URL first? Hmm.

Reply:
I kinda like a single newline. Twitter ends up replacing it with a box as well.